js/content-scripts/tvdb.js

```diff
@@ -0,0 +1,57 @@
+let overseerrContainer, imdbId, tmdbId, mediaType, mediaInfo;
+
+containerOptions.anchorElement = '#translations';
+containerOptions.containerClass = 'mb-2 py-2';
+containerOptions.badgeBackground = '#313131';
+
+const imdbRegex = /\/title\/(tt\d+)(?:\/|$).*/;
+
+mediaType = document.location.pathname.startsWith('/movies') ? 'movie' : 'tv';
+let matches = $("a[href*='/title/tt']", $('#series_basic_info')).attr( "href" ).match(imdbRegex);
```

Comment:
Overseerr relies on the TMDB id for its content. Here you are collecting the IMDb id but your are not using it.
In the `On other sites` section of TVDB you actually have a direct link to the TMDB page and can therefore use this link to get the TMDB id and directly call the `queryMedia` function, instead of searching the media based on the title first.
Could you edit this script to do so ?